ghidra-assistant/src/main/java/reva/RevaService.java

```diff
@@ -103,7 +101,7 @@ public void sendToReva(RevaMessage message) {
      * @throws RevaServerException if there is a problem communicating with ReVa
      */
     private RevaMessage getMessageFromServer() throws RevaServerException {
-        URI endpoint = revaServerBase.resolve("/project/" + currentProgram.getName() + "/message");
+        URI endpoint = revaServerBase.resolve("/project/" + currentProgram.getDomainFile().getName() + "/message");
```

Comment:
Reminder to update the README.md for this change.

Reply:
I should put this context here too: 
I ran into a problem where I had two files in a project, `swtor.exe` and `swtor.exe_2024xxyy`. I was working in `swtor.exe_2024xxyy` and `currentProgram.getName()` was returning `swtor.exe`, so the Ghidra plugin kept hitting the wrong `reverse-engineering-assistant` API endpoint. After exploring the Ghidra API for a while, I eventually found that `currentProgram.getDomainFile().getName()` would return the correct `swtor.exe_2024xxyy` name.